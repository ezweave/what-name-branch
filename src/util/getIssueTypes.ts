import { flow, values, toUpper } from 'lodash';
import { map as mapFP } from 'lodash/fp';

import { IssueType } from '@ezweave/types';

export const getIssueTypes = () => flow(
  values,
  mapFP(toUpper),
)(IssueType);


