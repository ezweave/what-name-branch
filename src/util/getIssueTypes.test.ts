import { getIssueTypes } from './getIssueTypes';

describe(getIssueTypes, () => {
  it('turns the IssueType enum into an array', () => {
    expect(getIssueTypes()).toMatchSnapshot();
  });
});
