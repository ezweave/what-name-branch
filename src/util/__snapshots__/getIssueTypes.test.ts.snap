// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getIssueTypes turns the IssueType enum into an array 1`] = `
Array [
  "BUG",
  "SPIKE",
  "STORY",
]
`;
